invite/test/invite_bot.test.ts

```diff
@@ -0,0 +1,302 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {mocked} from 'ts-jest/utils';
+
+import {Invite, InviteAction} from '../src/types';
+import {InvitationRecord} from '../src/invitation_record';
+import {InviteBot} from '../src/invite_bot';
+import {GitHub} from '../src/github';
+
+describe('Invite Bot', () => {
+  let inviteBot: InviteBot;
+
+  beforeEach(() => {
+    inviteBot = new InviteBot(/*client=*/ null, 'test_repo');
+
+    jest.spyOn(GitHub.prototype, 'inviteUser');
+    jest.spyOn(GitHub.prototype, 'addComment');
+    jest.spyOn(InvitationRecord.prototype, 'recordInvite');
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  describe('parseMacros', () => {
+    describe('comments without macros', () => {
+      it('returns an empty list', () => {
+        const comment = 'random comment';
+        expect(inviteBot.parseMacros(comment)).toEqual({});
+      });
+
+      it('ignores macros without usernames', () => {
+        const comment = '/invite not_a_tag';
+        expect(inviteBot.parseMacros(comment)).toEqual({});
+      });
+    });
+
+    describe('comments with `/invite @user`', () => {
+      it('returns list including Invite action for @user', () => {
+        const comment = '/invite @someone';
```

Comment:
Add a test for something like `meow/invite @someone` which should _not_ trigger

Reply:
Done